ipaserver/install/replication.py

```diff
@@ -515,9 +524,8 @@ def replica_config(self, conn, replica_id, replica_binddn):
             nsds5flags=["1"],
             nsds5replicabinddn=[replica_binddn],
             nsds5replicabinddngroup=[self.repl_man_group_dn],
-            nsds5replicabinddngroupcheckinterval=["60"],
-            nsds5replicareleasetimeout=["60"],
             nsds5replicalegacyconsumer=["off"],
+            **REPLICA_CREATION_SETTINGS
         )
```

Comment:
The overall patch looks good to me but there is something not clear to me.
ipa-replica-install creates two suffixes 'domain' and 'ipaca'. Those suffixes being shared between master and replica, ipa-replica-install also creates replication entry (cn=replica,cn=<suffix>,cn=mapping tree,cn=config).
It creates those entries on both master and replica.

My question, are those tuning applied on those entries on master host, replica host or on both ?

Reply:
Thanks that is perfect :)